app/javascript/controllers/map_controller.js

```diff
@@ -30,7 +30,7 @@ export default class extends Controller {
   }
 
   mapPositionOf(event) {
-    const { clientX, clientY } = event
+    const { clientX, clientY } = this.currentMapTarget.dataset
```

Comment:
This is also used to work out the the pointer location so probably better to build a custom event to pass in here instead.